src/Pages/propertyDetails/components/PropertyDetails.scss

```diff
@@ -0,0 +1,87 @@
+.PropertyDetails {
+
+  // image styles
+  #imageBlock {
+    background-color: #eee;
+    position: relative;
+    margin: 0;
+    display: table;
+    width: 40%;
+    padding: 10px;
+    height: 40%;
+    padding: 32px 0px;
+    background-color: white;
+    border-radius: 0px 0px 10px 10px ;
+    display: table;
+    width: 100%;
+    justify-content: center;
+    flex-direction: column;
+    
+
+  }
+
+  #imageBlock > img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    
+
+  }
+
+  // more info box- styling
+  .propertyDetailsBlock {
+    background-color: #81BDBE;
+    width: 100%;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    height: auto;
+    border: 6px solid #e7edf1;
+    text-align: justify;
+    padding-left: 5%;
+    padding-right: 5%;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+
+  // header styles
+  .propertyDetailsBlock > h2 {
+    background-color: #eee;
+    color: #010102;
+    padding: 20px;
+    border-radius: 6px;
+    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+    margin-bottom: 20px;
+    font-size: 38px;
+    text-align: center;
+  }
+  
+  .propertyDetailsBlock > h3 {
+    color: #447399;
```

Comment:
Dont think there is enough contrast here between the text and background, Pa11y will probs shout at you